<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoonOrbitPrefWidget</class>
 <widget class="QWidget" name="MoonOrbitPrefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>月球轨道</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>117</width>
     <height>88</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="mVisibilityBox">
      <property name="text">
       <string>显示轨道</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>线宽:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="mLineWidthBox">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>颜色:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="UtQtColorButton" name="mColorButton">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtColorButton</class>
   <extends>QPushButton</extends>
   <header>UtQtColorButton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
